src/legacy/status_im/ui/screens/advanced_settings/views.cljs

```diff
@@ -77,6 +78,15 @@
        [:wakuv2.ui/toggle-light-client (not light-client-enabled?)])
      :accessory :switch
      :active light-client-enabled?}
+    {:size :small
```

Comment:
should we put these settings in some new section?
cc @ilmotta as I think he has another pr cleaning up the old settings

Reply:
I think it was for Privacy & security, but if we have one for advanced, for sure

Reply:
Clean-up of the advanced settings should be done ideally, or at least some settings. I just haven't created one issue, but will do. For example, there's a setting for telemetry, but this is super confusing for the user as we already have in-app metrics now.